doc-content/enterprise-only/admin-and-config/managing-business-central-configuring-githooks-proc.adoc

```diff
@@ -140,16 +140,15 @@ end up being converted into a different code which may cause showing a wrong not
 ====
 
 == Internationalization support
-It is possible to internationalize the notification messages by placing different `*.properties` on the same path as the
-`*.properties` file specified on the `appformer.git.hooks.bundle` system property. The name of the different localized
+It is possible to internationalize the notification messages by placing different properties on the same path as the properties file specified on the `appformer.git.hooks.bundle` system property. The name of the different localized
```

Comment:
"You can internationalize the notification ...."

Reply:
Done